### PR TITLE
Initialize Project with Components

### LINKED ISSUE #5: Initialise Project
1. Create header
2. Create header and book list page

### PR DESCRIPTION
**Pull request by bushmusi**
> # Below features have added

- [x] 1. Initialise react app
- [x] 2. I have created a **component** `dir` for re-usable components
- [x] 3. Make book as default view
- [x] 4. Add `route link` to each page

> ** Helpfull links**

1. [Project requirment](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md)
2. [ Live Demo](https://bushmusi-bookstore.herokuapp.com/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
-      </header>
-    </div>
+
+    <Router>
+      <Header />
+      <div className="App">
+        <Routes>
+          <Route path="/book" element={<BookList />} />
+          <Route path="/catagory" element={<Catagory />} />
+          <Route path="/" element={<BookList />} />
```
Comment (elmar8287) [inline src/App.js]: - [ ] Please, consider that In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page:
```jsx
<Route exact path="/">
   <BookList />
</Route>
```

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @bushmusi.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- No linter issues 👍 
- Deployed demo version 👍 

## Required Changes ♻️

- [ ]  _Check the comments under the review._
- [ ] Please, fix in your README file the following section. The links are not related to the current project and the screenshot is also.
Screen Shot 2022-04-02 at 14 14 14

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (bushmusi) [reply src/App.js]: Hi @elmar8287 

You are right 👍 Noted and fixed

**Review by medaminedev66 — COMMENTED** [review]
## Hi @bushmusi, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**